packages/nodes-base/nodes/SyncroMSP/v1/actions/contact/add/description.ts

```diff
@@ -0,0 +1,109 @@
+import {
+	ContactProperties,
+} from '../../Interfaces';
+
+export const contactAddDescription: ContactProperties = [
+	{
+		displayName: 'Customer ID',
+		name: 'customerId',
+		type: 'string',
+		displayOptions: {
+			show: {
+				resource: [
+					'contact',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		default: '',
+	},
+	{
+		displayName: 'Email',
+		name: 'email',
+		type: 'string',
+		displayOptions: {
+			show: {
+				resource: [
+					'contact',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		default: '',
+	},
+	{
+		displayName: 'Name',
+		name: 'name',
+		type: 'string',
+		displayOptions: {
+			show: {
+				resource: [
+					'contact',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		default: '',
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		placeholder: 'Add Field',
+		displayOptions: {
+			show: {
+				resource: [
+					'contact',
+				],
+				operation: [
+					'add',
+				],
+			},
+		},
+		default: {},
+		options: [
+			{
+				displayName: 'Address',
+				name: 'address1',
```

Comment:
If we won't have multiple addresses inside n8n, simply use `address` as the variable name and map it accordingly when creating the payload. We try to avoid mixing the external services' naming issues with ours.